lms/djangoapps/experiments/views.py

```diff
@@ -84,3 +91,26 @@ class ExperimentKeyValueViewSet(viewsets.ModelViewSet):
     permission_classes = (IsStaffOrReadOnly,)
     queryset = ExperimentKeyValue.objects.all()
     serializer_class = serializers.ExperimentKeyValueSerializer
+
+
+class UserMetaDataView(APIView):
```

Comment:
Discussed over Slack - might be worth seeing how other endpoints guard access (maybe there's already a permission / method / pre-access-hook / etc.) so we can follow the same pattern.

Reply:
The closest-looking permission had been IsStaffOrOwner, but that's used by views that take a viewsets.ModelViewSet object, whereas ours takes an APIView (so we don't have an 'action' attribute that the permission expects). The codebase had several different 'setups' for doing API calls needing different things- I don't recall what  Matthew had said in our permissions pairing session about the different merits of each- I think we needed this one because it'll be an API returning JSON rather than a typical view- @MatthewPiatetsky is that right? Anything to add?